class.jetpack.php

```diff
@@ -3079,8 +3099,7 @@ public static function get_activation_source( $referer_url ) {
 	public static function do_version_bump( $version, $old_version ) {
 
 		if ( ! $old_version ) { // For new sites
-			// Setting up jetpack manage
-			Jetpack::activate_manage();
+			// There used to be stuff here, but this seems like it might  be useful to someone in the future...
```

Comment:
Lets remove it. Since we don't need it anymore. Should be easy enough to add.

Reply:
Remove the whole method `do_version_bump()`?

Reply:
I'd leave it for what comes after it (re-adding the connection banner) and provide a place to add any future things needed.